.github/workflows/lint.yml

```diff
@@ -12,13 +12,11 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
-      - uses: actions/setup-node@v3
-        with:
-          node-version: 18
-          cache: npm
+      - name: asdf_install
+        uses: asdf-vm/actions/install@v1
```

Comment:
https://github.com/kachick/kachick.github.io/actions/runs/3250444120/jobs/5334135118
https://github.com/asdf-vm/asdf-plugins/pull/691
https://github.com/asdf-vm/actions/blob/171cc3570019f3a69ba10ae269ab46990936fa9f/lib/plugins-add/index.ts#L31-L43
https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions#setting-an-output-parameter|